core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java

```diff
@@ -533,24 +534,31 @@ public Response getWebUILogs(@DefaultValue("") @QueryParam("path") String reques
   @Path(WEBUI_CONFIG)
   public Response getWebUIConfiguration() {
     return RestUtils.call(() -> {
-      WorkerWebUIConfiguration response = new WorkerWebUIConfiguration();
-      response.setWhitelist(mBlockWorker.getWhiteList());
+      MasterWebUIConfiguration response = new MasterWebUIConfiguration();
```

Comment:
`WorkerWebUIConfiguration`?

Reply:
Done

Reply:
interestingly these two are the same :( and can confuse ppl